build.gradle.kts

```diff
@@ -79,5 +80,13 @@ kotlin {
     }
 }
 
+detekt {
```

Comment:
Can you move it out to a new standalone script? Just like dokka.gradle.kts. That would help in having concise gradle files.

Reply:
that won't work 
> Script plugins do not have access to the main buildscript classpath.
![image](https://user-images.githubusercontent.com/20759720/94917808-f1f8b000-04b9-11eb-8dbc-a7d39e9c3384.png)
![image](https://user-images.githubusercontent.com/20759720/94917822-f755fa80-04b9-11eb-8dea-c07bac8dd8cd.png)

https://discuss.gradle.org/t/cant-access-classes-from-plugin-directly-in-included-script/14721
https://stackoverflow.com/questions/53739327/gradle-not-recognizing-plugin-tasks-from-applied-child-script

Reply:
Right. Hopefully https://github.com/detekt/detekt-compiler-plugin becomes stable soon.